libnmstate/metadata.py

```diff
@@ -54,13 +57,16 @@ def generate_ifaces_metadata(desired_state, current_state):
         get_slaves_func=linux_bridge.get_slaves_from_state,
         set_metadata_func=linux_bridge.set_bridge_ports_metadata
     )
+    _generate_route_metadata(desired_state)
 
 
 def remove_ifaces_metadata(ifaces_state):
     for iface_state in six.viewvalues(ifaces_state.interfaces):
         iface_state.pop('_master', None)
         iface_state.pop('_master_type', None)
         iface_state.pop('_brport_options', None)
+        iface_state.get(Interface.IPV4, {}).pop(Route.KEY, None)
```

Comment:
The other metadata is in a namespace with underscores... Maybe this should be considered here as well. On the other hand using these metadata fields seems to me to be just a workaround instead of making it a real objects.